include/wx/cmndata.h

```diff
@@ -160,6 +160,7 @@ class WXDLLIMPEXP_CORE wxPrintDialogData: public wxObject
     int GetMinPage() const { return m_printMinPage; }
     int GetMaxPage() const { return m_printMaxPage; }
     int GetNoCopies() const { return m_printNoCopies; }
+    int GetCurrentPage() const { return m_printCurrentPage; }
```

Comment:
This seems completely unused to me, as `m_printCurrentPage` is never set (nor used) anywhere. What is the point of having this function? IMO it should be just removed.

Reply:
Was not unsused, but my last commit has changed this too.